__tests__/app.test.js

```diff
@@ -60,12 +60,40 @@ describe("GET /api/topics ", () => {
         });
       });
   });
-  test("400: Respond with msg that topisc not found if we requesting a invalid route", () => {
+});
+
+describe.only("GET /api/articles/:article_id", () => {
+  test("200: Respond with one object from article table, and the object should have id that we passed in URL", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.article_id).toBe(1);
+        expect(body.article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: 1,
```

Comment:
You don't need to check the article_id is 1 twice, so you can remove one of the checks!